lotto.css

```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+html, body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    font-weight: 400;
+    font-family: Roboto;
+    font-size: 15px;
+    line-height: 24px;
+    min-height: 855px;
+    min-width: 414px;
+}
+
+/* 헤더,푸터 */
+header{
+    position: fixed;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    background-color: #4E5BA6;
+    height: 64px;
+    top: 0;
+}
+
+header p{
+    position: absolute;
+    color: white;
+    left: 130px;
+}
+
+
+footer{
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 80px;
+    border: 1px solid;  
+}
+
+footer p{
+    color:#4E5BA6;
+}
+
+
+
+/* 메인 */
+.main-contents{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    width: 414px;
+    height: 727px;
+    border: 1px solid #0000001F;   
+}
+
+.main-contents > * {
+    position: relative;
+    border-radius: 4px;
+    margin-top: 28px;
+    margin: 16px;
+}
+
+.main-contents input {
+    height: 36px;
+    border: 1px solid #0000001F;
+    box-sizing: border-box; 
+    font-size: 15px;
+    padding: 8px 8px;
+    margin-right: 3px;
+    border-radius: 4px;
+    border: 1px solid #B4B4B4
+}
+
+.main-contents button {
+    height: 36px;
+    background: #4E5BA6;
+    color: white;
+    border-radius: 4px;
+    border: none;
+}
+
+/* 금액입력폼 */
+.purchase-form{
+    width: 382px;
+    height: 64px;
+    margin-top: 28px;
+}
+
+.purchase-form input{
+    margin-top: 4px;
+    width: 310px;
+}
+
+.purchase-form button{
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    width: 56px;
+}
+
+/* 로또들 */
+.lotto-tickets-container{
+    width: 382px;
+    margin-top: 28px;
+}
+
+.lotto-tickets-container div{
+    margin-top: 4px;
+    letter-spacing: 0.5px;
+}
+
+.lotto-tickets-container > p{
+    margin-bottom: 8px;
+}
+
+/* 당첨번호입력란 */
+.winning-lotto-form{
+    margin-top: 24px;
+}
+.number-input-container{
+    position: relative;
+    margin-top: 12px;
+}
+.winning-lotto-form > div input{
+    width: 34px;
+    margin-top: 10px;
+}
+
+.bonus{
+    position: absolute;
+    right: 0;
+    top: 0;
+}
+
+.bonus input{
+    position: absolute;
+    right: 0;
+}
```

Comment:
위에서 쭉 보면서 느낀건, margin으로 간격을 만들고, absolute, p 태그를 많이 활용을 하는 것 같아요.
CSS의 flex를많이 활용해 보았으면 좋겠어요. 지금 ui를 보면 이렇게 밀려있단 말이죠?

![image](https://github.com/user-attachments/assets/463ce1f8-8a5b-48df-bf83-7433811f53e9)

이건 p 태그 때문에 생겨난 문제인데 이 문제는 위에서도 한 번 언급했다시피 해결 방법을 찾아보았으면 좋겠어요!

flex를 활용한다면 어떤 형식으로 짤 수 있는지 살펴보면 이 정도가 되겠네요!

```suggestion
.winning-lotto-form{
  display: flex;
  flex-direction: column;
  gap: 12px;
}

.number-input-container{
  display: flex;
  justify-content: space-between;
}

.number-input-container input{
  width: 36px;
}

.winning, .bonus {
  display: flex;
  flex-direction: column;
  gap: 8px;
}

.bonus {
  align-items: flex-end;
}

```